runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,155 @@
+{
+    "ociVersion": "1.0.1-dev",
```

Comment:
do we need the dev version?

Reply:
this was autogenerated when running the `runc spec` command

Reply:
+1 for using non-dev version.